api/defs/carouselOptions.schema.json

```
@@ -0,0 +1,61 @@
+{
+    "$schema": "https://raw.githubusercontent.com/lenra-io/ex_component_schema/beta/priv/static/draft-lenra.json",
+    "$id": "defs/carouselOptions.schema.json",
+    "title": "CarouselOptions",
+    "description": "Element of type CarouselOptions",
+    "type": "object",
+    "properties": {
+        "height": {
+            "type": "number"
+        },
+        "enlargeCenterPage": {
+            "type": "boolean",
+            "default": false
+        },
+        "autoPlay": {
+            "type": "boolean",
+            "default": false
+        },
+        "autoPlayInterval": {
+            "$ref": "defs/duration.schema.json"
```

The path must be relatve:
```suggestion
            "$ref": "../defs/duration.schema.json"
```